### PR TITLE
Made review changes suggested by Hashicorp

### PR DESCRIPTION
**Pull request by saloneerege**

### CODE REVIEW

**Review by shylasrinivas — COMMENTED** [review]
Apart from README (for which I have some comments), rest looks good.

- We had purposely removed instructions on how to use the provider in the README since Go SDK is not public yet. Do we want to add them back for the sake of the review or comment them out or let the Hashicorp reviewer know why the README does not look complete?
- References to main.tf can be changed to examples/main.tf

File: .gitignore [hunk 1]
```diff
@@ -9,4 +9,5 @@ terraform.tfstate
 *.backup
 *.bak
 *~
-.terraform.tfstate.lock.info
\ No newline at end of file
+.terraform.tfstate.lock.info
+*.tf
```
Comment (rchadalawada) [inline .gitignore]: update this to ignore only the .tf files in the root directory

File: README.md [hunk 2]
```diff
@@ -3,16 +3,149 @@
...
 Terraform to work with [VMware Cloud on AWS](https://vmc.vmware.com/).
 
-**This repository is Work in Progress and has been made public only for review purposes.**
+# Using the provider
 
 Requirements
 ------------
 
 - [Terraform](https://www.terraform.io/downloads.html) 0.12+
 - [Go](https://golang.org/doc/install) 1.13 (to build the provider plugin)
```
Comment (rchadalawada) [inline README.md:12]: Should we specify that these instructions are for mac or linux only

File: README.md [hunk 3]
```diff
@@ -3,16 +3,149 @@
...
+cd $GOPATH/src/github.com/provider/
+git clone https://github.com/vmware/terraform-provider-vmc.git
+```
+
+Enter the provider directory and build the provider
+
+```sh
+cd $GOPATH/src/github.com/provider/terraform-provider-vmc
+go get
+go build -o terraform-provider-vmc
```
Comment (rchadalawada) [inline README.md:30]: should we instead say "make"? Also, we should fix the 'go fmt' error that we are getting when we do just "make"
rchadalawad-a01:tpv rchadalawada$ make
==> Checking that code complies with gofmt requirements...
gofmt needs running on the following files:
./vmc/provider.go
You can use the command: `make fmt` to reformat code.
make: *** [fmtcheck] Error 1
rchadalawad-a01:tpv rchadalawada$ make fmt

File: README.md [hunk 4]
```diff
@@ -3,16 +3,149 @@
...
+```sh
+terraform init
+```
+
+## Connect to VMC and create a SDDC
+
+Update following fields in the [main.tf](main.tf) with your infrastructure settings
+
+* refresh_token
+* id
```
Comment (rchadalawada) [inline README.md]: Org id

File: README.md [hunk 5]
```diff
@@ -3,16 +3,149 @@
...
+* sddc_name
+
```
Comment (rchadalawada) [inline README.md]: how about "sddc region"

Comment (rchadalawada) [reply README.md]: Actually, we should move these instructions to the examples README.md and just add a reference to that here

**Review by rchadalawada — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (saloneerege) [reply .gitignore]: This change was in another PR. I will rebase with master

File: README.md (hunk 2 above)
Comment (saloneerege) [reply README.md:12]: Yes I think we should.

Comment (rchadalawada) [reply README.md:12]: Please add a line stating that

**Pull request closed without merging**